dashboard/scripts/backfill_section_start.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+require_relative '../config/environment'
```

Comment:
Though I'm fine with this living in `dashboard/scripts/`, I've been dumping such scripts in `bin/oneoff/backfill_data`.